tests/lodash-deep-extras-tests.js

```diff
@@ -289,232 +228,165 @@ describe(
 
```

Comment:
do you mean the space above `beforeEach`? I went with the following rule: new line above `describe` `beforeEach` `afterEach`... effectively, just no new line above `it`